python/sbp/client/loggers/json_logger.py

```diff
@@ -33,9 +33,9 @@ def fmt_msg(self, msg):
       data = self.dispatcher(msg).to_json_dict()
     except KeyError:
       data = msg.to_json_dict()
-    return {"delta": self.delta(),
-            "timestamp": self.timestamp(),
-            "data": data}
+    msg = {"delta": self.delta(), "timestamp": self.timestamp(), "data": data}
```

Comment:
Hmmm, I think there's a risk here of tags stamping over some existing keys (`delta`, `timestamp`, `data`).